core/chains/evm/config/toml/defaults/fallback.toml

```
@@ -26,6 +26,22 @@ ResendAfterThreshold = '1m'
 [BalanceMonitor]
 Enabled = true
 
+[ClientErrors]
```

Would this make sense under `[NodePool]`?
```suggestion
[NodePool.Errors]
```